diff --git a/node_modules/react-native-maps/android/build.gradle b/node_modules/react-native-maps/android/build.gradle
index cc44531..0793612 100644
--- a/node_modules/react-native-maps/android/build.gradle
+++ b/node_modules/react-native-maps/android/build.gradle
@@ -29,6 +29,10 @@ android {
     namespace "com.rnmaps.maps"
   }
 
+  buildFeatures {
+    buildConfig true
+  }
+
   compileSdk safeExtGet('compileSdkVersion', 34)
 
   defaultConfig {
